### ISSUE TITLE
Checking for update Update for version 1.7.8 is not available (latest version: 1.7.8, downgrade is disallowed).

### ISSUE THREAD

**Issue by Firestar-Reimu** [balena-io/etcher#3743]
- **Etcher version: 1.7,8** 
- **Operating system and architecture: Manjaro Linux** 

When I running balena-etcher, it continuously say: 

`Checking for update
Update for version 1.7.8 is not available (latest version: 1.7.8, downgrade is disallowed).`

**Comment by mcraa** [comment]
And what it says is correct. 
What is your actual issue?

**Comment by Firestar-Reimu** [comment]
As this is the latest version, “Update for version 1.7.8 is not available” is not necessary.

**Comment by mcraa** [comment]
yes @thedocbwarren .9 was meaningful for debian based distros and the notification was not set to be sent to the users.
See stagingpercentage in [latest*.yml](https://github.com/balena-io/etcher/releases/tag/v1.7.9) files.

However this issue was opened before .9 was out and at the time .8 was the latest version.
